spec/values/calculation/round.hrx

```diff
@@ -0,0 +1,422 @@
+<===> up/negative_zero/input.scss
+@use "sass:math";
+a {b: math.div(1, round(up, -10, infinity))}
+
+<===> up/negative_zero/output.css
+a {
+  b: calc(-infinity);
+}
+
+<===>
+================================================================================
+<===> nearest/negative_zero/input.scss
+@use "sass:math";
+a {b: math.div(1, round(nearest, -10, infinity))}
+
+<===> nearest/negative_zero/output.css
+a {
+  b: calc(-infinity);
+}
+
+<===>
+================================================================================
+<===> down/negative_zero/input.scss
+@use "sass:math";
+a {b: math.div(1, round(down, -0, infinity))}
+
+<===> down/negative_zero/output.css
+a {
+  b: calc(-infinity);
+}
+
+<===>
+================================================================================
+<===> operation/positive/input.scss
+a {b: round(up, 10, infinity)}
+
+<===> operation/positive/output.css
+a {
+  b: calc(infinity);
+}
+
+<===>
+================================================================================
+<===> operation/negative/input.scss
+@use "sass:math";
+a {b: round(down, -10, infinity)}
+
+<===> operation/negative/output.css
+a {
+  b: calc(-infinity);
+}
+
+<===>
+================================================================================
+<===> operation/one_arg/simplify/input.scss
+a {b: round(3.8px + 4px)}
+
+<===> operation/one_arg/simplify/output.css
+a {
+  b: 8px;
+}
+
+<===>
+================================================================================
+<===> operation/two_args/simplify/input.scss
+$c: 10px;
+a {b: round(3.8px + 4px, $c)}
+
+<===> operation/two_args/simplify/output.css
+a {
+  b: 10px;
+}
+
+<===>
+================================================================================
+<===> operation/nearest/simplify/input.scss
+a {b: round(nearest, 115px + 2px, 21px + 4px)}
+
+<===> operation/nearest/simplify/output.css
+a {
+  b: 125px;
+}
+
+<===>
+================================================================================
+<===> operation/up/input.scss
+a {b: round(up, 101px, 25px)}
+
+<===> operation/up/output.css
+a {
+  b: 125px;
+}
+
+<===>
+================================================================================
+<===> operation/down/input.scss
+a {b: round(down, 122px, 25px)}
+
+<===> operation/down/output.css
+a {
+  b: 100px;
+}
+
+<===>
+================================================================================
+<===> operation/to-zero/positive/input.scss
+a {b: round(to-zero, 120px, 25px)}
+
+<===> operation/to-zero/positive/output.css
+a {
+  b: 100px;
+}
+
+<===>
+================================================================================
+<===> operation/to-zero/negative/input.scss
+a {b: round(to-zero, -120px, 25px)}
+
+<===> operation/to-zero/negative/output.css
+a {
+  b: -100px;
+}
+
+<===>
+================================================================================
+<===> NaN/simplify/input.scss
```

Comment:
The test directories should probably be all lower case to avoid any potential issues with case-sensitive vs case-insensitive file systems.

@nex3: Is this something we should add to the style guide?